src/pages/contact-us/_offices.js

```diff
@@ -149,7 +149,7 @@ export const Offices = () => {
                                     target="_blank"
                                     color="black-3"
                                     size="var(--text-size-s)"
-                                    href="https://maps.google.com/?q=Binary+(Europe)+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
+                                    href="https://maps.google.com/?q=Binary+Europe+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
```

Comment:
why does this make any difference?

Reply:
Really don't know!!! but it didn't work like that!